### PR TITLE
Pioneer 20b gliders

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: deployment/CP15MOAS-G0913_Deploy.csv [hunk 1]
```diff
@@ -5,3 +5,9 @@ AR82,,AR82,,CP15MOAS-G0913-02-FLORTM000,1,1,2024-04-05T18:06:00,2024-04-05T21:43
...
 AR82,,AR82,,CP15MOAS-G0913-05-PARADM000,1,1,2024-04-05T18:06:00,2024-04-05T21:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-PARADM-50333,35.88125,-74.83379,,200,200,CS,,
+PH20240630,,,,CP15MOAS-G0913-00-ENG000000,2,1,2024-06-30T15:43:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGCON-GLDRCP-00913,35.96352,-74.79357,,200,200,CS,,
+PH20240630,,,,CP15MOAS-G0913-01-ADCPAM000,2,1,2024-06-30T15:43:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-ADCPAM-73245,35.96352,-74.79357,,200,200,CS,,
+PH20240630,,,,CP15MOAS-G0913-02-FLORTM000,2,1,2024-06-30T15:43:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-FLORTM-06642,35.96352,-74.79357,,200,200,CS,,
+PH20240630,,,,CP15MOAS-G0913-03-CTDGVM000,2,1,2024-06-30T15:43:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-CTDGVM-09690,35.96352,-74.79357,,200,200,CS,,
+PH20240630,,,,CP15MOAS-G0913-04-DOSTAM000,2,1,2024-06-30T15:43:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-DOSTAM-00909,35.96352,-74.79357,,200,200,CS; top mount,,
+PH20240630,,,,CP15MOAS-G0913-05-PARADM000,2,1,2024-06-30T15:43:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-PARADM-50333,35.96352,-74.79357,,200,200,CS,,
```
Comment (rtravis-whoi) [inline deployment/CP15MOAS-G0913_Deploy.csv:13]: looks good

File: deployment/CP15MOAS-GL379_Deploy.csv [hunk 2]
```diff
@@ -10,4 +10,10 @@ KH20240523,,KH20240609,,CP15MOAS-GL379-01-ADCPAM000,2,1,2024-05-23T13:32:00,2024
...
\ No newline at end of file
+KH20240523,,KH20240609,,CP15MOAS-GL379-05-PARADM000,2,1,2024-05-23T13:32:00,2024-06-09T14:27:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50220,35.92208,-74.75223,,1000,1000,OM,,
+PH20240630,,,,CP15MOAS-GL379-00-ENG000000,3,1,2024-06-30T13:08:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,35.95345,-74.76803,,1000,1000,OM,,
+PH20240630,,,,CP15MOAS-GL379-01-ADCPAM000,3,1,2024-06-30T13:08:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49016,35.95345,-74.76803,,1000,1000,OM,,
+PH20240630,,,,CP15MOAS-GL379-02-FLORTM000,3,1,2024-06-30T13:08:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-04194,35.95345,-74.76803,,1000,1000,OM,,
+PH20240630,,,,CP15MOAS-GL379-03-CTDGVM000,3,1,2024-06-30T13:08:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09030,35.95345,-74.76803,,1000,1000,OM,,
+PH20240630,,,,CP15MOAS-GL379-04-DOSTAM000,3,1,2024-06-30T13:08:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00496,35.95345,-74.76803,,1000,1000,OM; top mount,,
+PH20240630,,,,CP15MOAS-GL379-05-PARADM000,3,1,2024-06-30T13:08:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50220,35.95345,-74.76803,,1000,1000,OM,,
```
Comment (rtravis-whoi) [inline deployment/CP15MOAS-GL379_Deploy.csv:19]: looks good

File: deployment/CP15MOAS-GL387_Deploy.csv [hunk 3]
```diff
@@ -10,4 +10,10 @@ KH20240523,,KH20240616,,CP15MOAS-GL387-01-ADCPAM000,2,1,2024-05-23T15:08:00,2024
...
\ No newline at end of file
+KH20240523,,KH20240616,,CP15MOAS-GL387-05-PARADM000,2,1,2024-05-23T15:08:00,2024-06-16T18:42:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-PARADM-50151,35.94993,-74.79575,,350,350,MA,,
+PH20240630,,,,CP15MOAS-GL387-01-ADCPAM000,3,1,2024-06-30T15:40:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGCON-GLDRCP-00387,35.96227,-74.7948,,350,350,MA,,
+PH20240630,,,,CP15MOAS-GL387-02-FLORTM000,3,1,2024-06-30T15:40:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-ADCPAM-55979,35.96227,-74.7948,,350,350,MA,,
+PH20240630,,,,CP15MOAS-GL387-03-CTDGVM000,3,1,2024-06-30T15:40:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-FLORTM-04471,35.96227,-74.7948,,350,350,MA,,
+PH20240630,,,,CP15MOAS-GL387-04-DOSTAM000,3,1,2024-06-30T15:40:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-CTDGVM-09059,35.96227,-74.7948,,350,350,MA,,
+PH20240630,,,,CP15MOAS-GL387-05-PARADM000,3,1,2024-06-30T15:40:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-DOSTAM-00310,35.96227,-74.7948,,350,350,MA; top mount,,
+PH20240630,,,,CP15MOAS-GL387-05-PARADM000,3,1,2024-06-30T15:40:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-PARADM-50151,35.96227,-74.7948,,350,350,MA,,
```
Comment (rtravis-whoi) [inline deployment/CP15MOAS-GL387_Deploy.csv:19]: looks good

File: deployment/CP15MOAS-PG564_Deploy.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+PH20240630,,,,CP15MOAS-PG564-00-ENG000000,1,1,2024-06-30T13:27:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00564,CGCON-GLDRCP-00564,35.95762,-74.7631,,1000,1000,OF,,
+PH20240630,,,,CP15MOAS-PG564-01-CTDGVM000,1,1,2024-06-30T13:27:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00564,CGINS-CTDGVM-09307,35.95762,-74.7631,,1000,1000,OF,,
+PH20240630,,,,CP15MOAS-PG564-02-DOSTAM000,1,1,2024-06-30T13:27:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00564,CGINS-DOSTAM-00604,35.95762,-74.7631,,1000,1000,OF; top mount,,
+PH20240630,,,,CP15MOAS-PG564-03-FLORTM000,1,1,2024-06-30T13:27:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00564,CGINS-FLORTM-04093,35.95762,-74.7631,,1000,1000,OF,,
+PH20240630,,,,CP15MOAS-PG564-04-FLORTO000,1,1,2024-06-30T13:27:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00564,CGINS-FLORTO-01348,35.95762,-74.7631,,1000,1000,OF,,
+PH20240630,,,,CP15MOAS-PG564-05-NUTNRM000,1,1,2024-06-30T13:27:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00564,CGINS-NUTNRM-00658,35.95762,-74.7631,,1000,1000,OF,,
+PH20240630,,,,CP15MOAS-PG564-06-PARADM000,1,1,2024-06-30T13:27:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00564,CGINS-PARADM-50218,35.95762,-74.7631,,1000,1000,OF,,
```
Comment (rtravis-whoi) [inline deployment/CP15MOAS-PG564_Deploy.csv:8]: looks good

**Review by rtravis-whoi — APPROVED** [review]

**Comment by cdobs** [comment]
FCPs: 379: 3304-00150-00171
387: 3304-00150-00169
564: 3304-00180-00011
913: 3304-00150-00172

File: deployment/CP15MOAS-PG564_Deploy.csv (hunk 4 above)
Comment (cooleyky) [inline deployment/CP15MOAS-PG564_Deploy.csv:8]: looks good

File: deployment/CP15MOAS-GL379_Deploy.csv (hunk 2 above)
Comment (cooleyky) [inline deployment/CP15MOAS-GL379_Deploy.csv:19]: looks good

File: deployment/CP15MOAS-GL387_Deploy.csv [hunk 5]
```diff
@@ -10,4 +10,10 @@ KH20240523,,KH20240616,,CP15MOAS-GL387-01-ADCPAM000,2,1,2024-05-23T15:08:00,2024
...
\ No newline at end of file
+KH20240523,,KH20240616,,CP15MOAS-GL387-05-PARADM000,2,1,2024-05-23T15:08:00,2024-06-16T18:42:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-PARADM-50151,35.94993,-74.79575,,350,350,MA,,
+PH20240630,,,,CP15MOAS-GL387-00-ENG000000,3,1,2024-06-30T15:40:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGCON-GLDRCP-00387,35.96227,-74.7948,,350,350,MA,,
+PH20240630,,,,CP15MOAS-GL387-01-ADCPAM000,3,1,2024-06-30T15:40:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-ADCPAM-55979,35.96227,-74.7948,,350,350,MA,,
+PH20240630,,,,CP15MOAS-GL387-02-FLORTM000,3,1,2024-06-30T15:40:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-FLORTM-04471,35.96227,-74.7948,,350,350,MA,,
+PH20240630,,,,CP15MOAS-GL387-03-CTDGVM000,3,1,2024-06-30T15:40:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-CTDGVM-09059,35.96227,-74.7948,,350,350,MA,,
+PH20240630,,,,CP15MOAS-GL387-04-DOSTAM000,3,1,2024-06-30T15:40:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-DOSTAM-00310,35.96227,-74.7948,,350,350,MA; top mount,,
+PH20240630,,,,CP15MOAS-GL387-05-PARADM000,3,1,2024-06-30T15:40:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-PARADM-50151,35.96227,-74.7948,,350,350,MA,,
```
Comment (cooleyky) [inline deployment/CP15MOAS-GL387_Deploy.csv:19]: looks good

File: deployment/CP15MOAS-G0913_Deploy.csv (hunk 1 above)
Comment (cooleyky) [inline deployment/CP15MOAS-G0913_Deploy.csv:13]: looks good

**Review by cooleyky — APPROVED** [review]
looks good

**Pull request merged**